app/assets/javascripts/discourse/widgets/post.js.es6

```diff
@@ -365,6 +365,27 @@ createWidget("post-contents", {
     ];
     result = result.concat(applyDecorators(this, "after-cooked", attrs, state));
 
+    if (
+      attrs.membershipRequest &&
+      this.currentUser.owned_groups &&
+      this.currentUser.owned_groups.includes(attrs.membershipRequest)
```

Comment:
Are we able to figure out if we should show the membership request link in the `TopicView` serializer instead? The worry I have here is that the membership request links are only for PMs but we're adding additional fields to the current user payload for every page load.